test/acceptance/features/postgresqlOperatorExamples.feature

```diff
@@ -0,0 +1,331 @@
+        * DB "db-demo-a-d-c" is running
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-a-d-c
+            spec:
+                application:
+                    name: nodejs-rest-http-crud-a-d-c
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: db-demo-a-d-c
+                    id: postgresDB
+                mappings:
+                    - name: SOME_KEY
+                      value: 'SOME_VALUE:{{ .postgresDB.status.dbConnectionPort }}:{{ .postgresDB.status.dbName }}'
+            """
+        Then Service Binding "binding-request-a-d-c" is ready
+        And Secret contains "SOME_KEY" key with value "SOME_VALUE:5432:db-demo-a-d-c"
+
+    Scenario: Bind two backend services by creating 2 SBRs to a single application
+        Given Imported Nodejs application "nodejs-app" is running
+        * DB "db-demo-1" is running
+        * DB "db-demo-2" is running
+        * Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-1
+            spec:
+                application:
+                    name: nodejs-app
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: db-demo-1
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-2
+            spec:
+                application:
+                    name: nodejs-app
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: db-demo-2
+            """
+        Then Service Binding "binding-request-1" is ready
+        And Service Binding "binding-request-2" is ready
+        And "nodejs-app" deployment must contain SBR name "binding-request-1"
+        And "nodejs-app" deployment must contain SBR name "binding-request-2"
+
+    # https://github.com/redhat-developer/service-binding-operator/tree/master/examples/pod_spec_path
+    Scenario: Specify container's path in Service Binding
```

Comment:
do we have now the same scenario but without database available in some other feature file?

Reply:
Yes we've at https://github.com/redhat-developer/service-binding-operator/pull/880/files#diff-ca848b998fbd78a0989748822c09065858004a5e79b457ae9b57ad5ed9dcd25cR117